rpcs3/Emu/SysCalls/Modules/cellAvconfExt.cpp

```diff
@@ -114,6 +144,9 @@ s32 cellVideoOutGetScreenSize(u32 videoOut, vm::ptr<float> screenSize)
 
 Module cellAvconfExt("cellAvconfExt", []()
 {
+	fxm::remove<avconfext_t>();
```

Comment:
Hopefully fixed this. If it's still wrong, please point out, what I'm doing wrong.